### ISSUE TITLE
Killed before Qt disconnects from X with pytest-qt?

### ISSUE THREAD

**Issue by The-Compiler** [The-Compiler/pytest-xvfb#1]
It seems https://github.com/pytest-dev/pytest-qt/pull/124 is [causing some trouble](https://travis-ci.org/pytest-dev/pytest-qt/builds/109691139):

```
=================== 123 passed, 2 skipped in 17.15 seconds ====================
plugin.py: Fatal IO error: client killed
```

It smells like Xvfb gets terminated while Qt is still connected to it?
- https://github.com/thoughtbot/capybara-webkit/issues/356
- http://stackoverflow.com/questions/17712301/xvfb-io-error-client-killed
- http://stackoverflow.com/questions/7116635/capybara-webkit-rspec-integration-specs-and-xvfb-webkit-server-fatal-io-erro

**Comment by ionelmc** [comment]
Maybe QT don't disconnect cleanly?

**Comment by The-Compiler** [comment]
Dang, I thought this would be fixed by #9, but apparently not :wink:

**Comment by The-Compiler** [comment]
I've not seen this happen anymore in the last couple of years.
